resources/language/resource.language.hu_hu/strings.po

```diff
@@ -939,3 +941,7 @@ msgstr "Nézettként jelöl|Nem látottként jelöl|Netflix nézett státusz vis
 msgctxt "#30238"
 msgid "Up Next notifications (watch next video)"
 msgstr "Up Next jelzés (következő videó nézése)"
+
+msgctxt "#30239"
+msgid "Color of supplemental plot info"
+msgstr "A kiegészítő információ színe"
```

Comment:
i think you have to explain what info,
see discussion and screenshot in to https://github.com/CastagnaIT/plugin.video.netflix/pull/568
I could also improve the sentence in English if it is a problem, just reach an agreement with everyone

Reply:
Hi, the translation correct, I have read that earlier... You can merge.